parent/pom.xml

```diff
@@ -502,7 +502,7 @@ Bundle-Category: jackrabbit
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-collections4</artifactId>
-                <version>4.1</version>
```

Comment:
Why is that upgrade necessary? That makes it incompatible with older distributions like AEM 6.4.

Reply:
FV is already incompatible with AEM 6.4 (unless I'm missing something).

Reply:
Really? At least from a dependency pov this is checked during the build. Why do you think otherwise?

Reply:
Because there are incompatibilties in AEM code that require code changes to be compatible with Filevault (newer than 3.4.0). You can probably deploy it, but there may be hard-to-identify problems later on.

Reply:
FWIW, see https://helpx.adobe.com/support/programs/eol-matrix.html - AEM 6.4 will be out of extended support in five months from now.

Reply:
It is not necessarily about FileVault Core but other bundles like RCP.

Reply:
Hmmm. RCP doesn't use commons-collections4 anyway, right?

In any case I agree that getting rid of the dependency would be preferable.